difficult_stage_1/script.js

```diff
@@ -0,0 +1,76 @@
+const audioSequence = [{qn:"which country was Alan Turing born in?", audioClip:"source/qn1 (1).mp3"}, {qn:"When did Bill Gates make the first computer?", audioClip:"source/qn2 (1).mp3"}, {qn:"What is the first mobile App in the world?", audioClip:"source/qn3 (1).mp3"},{qn:"What is the most popular computer language now?", audioClip:"source/qn4 (1).mp3"}, {qn:"What were Steve Jobs last designs?", audioClip:"source/qn5 (1).mp3"}]
+let index = 0
+
+function loadPage(){
+      //display audio
+  audioDisplay()
+      //display qn
+  qn = document.getElementById('qn')
+  qn.textContent = audioSequence[index].qn
+  resizeInput()
+  console.log(index)
+}
+
+function audioDisplay() {
+  container = document.getElementById('audio')
+  var x = document.createElement('AUDIO')
+  x.canPlayType = 'audio/mpeg'
+  x.setAttribute('src', audioSequence[index].audioClip)
+  x.setAttribute('controls', 'controls')
+  container.appendChild(x)
+}
+
+
+function toServer() {
+  userInput = document.getElementById('input').value
+  fetch('https://stem-week-cipher.herokuapp.com/testPicture', {
+    method: 'post',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({ans:userInput}), //where data is a json object.
+    credentials: 'include'
+   }
+  )
+  .then(function(response){
+    const contentType = response.headers.get("content-type")
+    if(contentType == 'image/png') {
+      return response.blob()
+    }else{
+      wrongAns = document.getElementById('wrong')
+      wrongAns.textContent = "oops, try again :D"
```

Comment:
```suggestion
      wrongAns.textContent = "oops, try again :D"
      return null;
```

you should return `null` or some value when the response type is not image else it will raise and error as you cant `urlCreator.createObjectURL(data)` when data is a JSON object.